libs/lib-client/core/src/services/restapi/v2/contributions.service.ts

```diff
@@ -23,6 +23,10 @@ export class ContributionsService {
     return this.restApi.get<T[]>(this.getApiPath() + '?filter[shim]=' + true);
   }
 
+  listAllContribs<T extends ContributionSchema = ContributionSchema>() {
```

Comment:
could we merge this with `listContribs()` below? We can just omit the `filter[type]` parameter  if `type` argument is not passed to `listContrib`.